src/main.py

```
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
+
+import os
```

Ruff (0.8.0)

`os` imported but unused

Remove unused import: `os`

(F401)